mappings/net/minecraft/client/gui/screen/DirectConnectScreen.mapping

```diff
@@ -1 +1,7 @@
 CLASS net/minecraft/class_738 net/minecraft/client/gui/screen/DirectConnectScreen
+	FIELD field_3144 parent Lnet/minecraft/class_754;
+	FIELD field_3145 info Lnet/minecraft/class_910;
+	FIELD field_3146 search Lnet/minecraft/class_689;
```

Comment:
Would want to suffix all these so it becomes `searchField`

Reply:
Sure I'll do that a few hours later